mk/travis.sh

```diff
@@ -88,7 +88,9 @@ fi
 case $TARGET_X in
 armv7-linux-androideabi)
   cargo test -vv -j2 --no-run ${mode-} ${FEATURES_X-} --target=$TARGET_X
```

Comment:
Please add a comment about why we're doing this here.

Also, would it be possible/beneficial for us to create the AVD once, have Travis CI cache it, and then make a copy of the cached AVD that we then run the tests on?